unique.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+int main(){
```

Comment:
It was a little strange to me that you didn't have it print out that it was unique when it was, but nevertheless, your code worked as needed!